client/app/models/project.js

```diff
@@ -120,6 +121,18 @@ export default class ProjectModel extends Model {
 
   @attr() videoLinks;
 
+  @attr() dcpApplicability;
+
+  @computed('dcpApplicability')
+  get dcpApplicabilitySimp() {
+    const option = Object.values(DCP_APPLICABILITY_OPTIONSET).find(x => x.code === this.dcpApplicability);
+    if (option) {
+      return option.label;
+    }
+
+    return 'Unknown';
```

Comment:
I wonder why we are returning `Unknown` instead of null. WIll that string value be saved to the sharepoint data?

Reply:
No, it won't be saved.  It is just in case the information is pulled up in the UI somewhere else, so what will be displayed as "Unknown" instead of "null".

Reply:
oh, yea, that makes sense. Ah, so the code that is saved would be from the `DCP_APPLICABILITY_OPTIONSET` const?